blind75/two_sum.py

```diff
@@ -59,6 +59,17 @@ def solution3(self, nums: List[int], target: int) -> List[int]:
                 if nums[i] + nums[j] == target:
                     return [i,j]
         return []
+    
+    # Time complexity: O(n2), Space complexity: O(1)
+    def solution3(self, nums: List[int], target: int) -> List[int]:
```

Comment:
Integrate the empty list check into the existing `solution3` method to avoid redundancy. This will maintain code cleanliness and reduce duplication. [important]